app/services/open_library_service.rb

```diff
@@ -0,0 +1,43 @@
+class OpenLibraryService
+  include HTTParty
+  base_uri 'https://openlibrary.org'
+
+  def initialize(isbn)
+    @isbn = isbn
+    @options = { query: { bibkeys: "ISBN:#{isbn}", format: 'json', jscmd: 'data' } }
+  end
+
+  def book
+    response = self.class.get('/api/books', @options)
+    format(@isbn, response)
+  end
+
+  private
+
+  def format(isbn, response)
```

Comment:
Hum I am not fan of the name, could be confusing and generate conflict, try something more descriptive